tests/envs/mujoco/test_mujoco_custom_env.py

```diff
@@ -107,5 +107,5 @@ def test_frame_skip(frame_skip):
 
 def test_xml_file():
     """Verify that the loading of a custom XML file works"""
-    env = PointEnv(xml_file="./envs/mujoco/assets/walker2d_v5_uneven_feet.xml")
+    env = PointEnv(xml_file="./assets/walker2d_v5_uneven_feet.xml")
```

Comment:
Is it possible to test the absolute path?

Reply:
Done,
I can not test the '~' with the GitHub CI though (there is no home folder)